pkg/translator/azure/resourcelogs_to_logs.go

```diff
@@ -47,6 +47,7 @@ type azureRecords struct {
 // https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/resource-logs-schema
 type azureLogRecord struct {
 	Time              string       `json:"time"`
+	Timestamp         string       `json:"timeStamp"`
```

Comment:
```suggestion
	Timestamp         string       `json:"timestamp"`
```
Timestamp is one word, so we don't want `Stamp` capitalized, right? Let me know if there's some precedent I'm not aware of here though.

Reply:
It looks like `timeStamp` may be an [Azure-specific format](https://learn.microsoft.com/en-us/azure/application-gateway/application-gateway-diagnostics#access-log). If so, please feel free to disregard.

Reply:
Yes, I just used what was shown in the example.